src/core_plugins/kibana/public/dashboard/styles/index.less

```diff
@@ -2,6 +2,44 @@
 @import (reference) "~ui/styles/mixins";
 @import "~ui/styles/local_search.less";
 
+.fullScreenModePlaceholder {
+  text-align: center;
+  width: 100%;
+  z-index: 50;
+  height: 0;
+  bottom: 0;
+  position: absolute;
+}
+
+.exitFullScreenMode {
+  background-color: @kibanaGray4;
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px;
+  border-bottom-left-radius: 0;
+  border-bottom-right-radius: 0;
+  width: 200px;
+  height: 40px;
+  left: 0;
+  right: 0;
+  margin-left: auto;
+  margin-right: auto;
+  padding-top: 0;
+
+  position: fixed;
+  bottom: -28px;
+  z-index: 50;
+  display: block;
+
+  .kuiIcon {
+    display: block;
+  }
+}
+
+.exitFullScreenMode:hover {
+  bottom: 0px;
+  transition: all .5s ease;
```

Comment:
Here's a cleaner animation for you...


![](https://d3vv6lp55qjaqc.cloudfront.net/items/3V1h382D1q181q2A3B01/Screen%20Recording%202017-06-20%20at%2002.54%20PM.gif?X-CloudApp-Visitor-Id=59773&v=cbfb8c24)

Does the following

* Moves the bug to the top, which is where the controls originally start.
* Uses more reliable vars.
* Improves the transition by
    * Using transform instead of position (much smoother, gpu now takes over).
    * Puts transition on the base selector, now works on on and off hover, instead of just on.
    * Speeds it up. In general, no hover animation should take longer than 250ms.
    * When in doubt use ease-in-out, rather than just ease.

*Note that you should also up the padding to `.dashboard-grid` to 20px if you move it up. In general I've been adding similar increased padding across Kibana, so it's not just for this PR that we need it.

```
.exitFullScreenMode {
  background-color: @globalColorLightestGray;
  border-bottom-left-radius: 4px;
  border-bottom-right-radius: 4px;
  border-top-left-radius: 0;
  border-top-right-radius: 0;
  width: 200px;
  height: 40px;
  left: 0;
  right: 0;
  top: 0;
  margin-left: auto;
  margin-right: auto;
  padding-bottom: 0;
  position: fixed;
  transform: translateY(-26px);
  z-index: 50;
  display: block;
  transition: transform .2s ease-in-out;
  border: solid 1px @globalColorLightGray;

  .kuiIcon {
    display: block;
  }

  &:hover {
    transform: translateY(-1px);
  }
}
```